packages/esm-ward-app/src/ward-workspace/admit-patient-form-workspace/admit-patient-form.test.tsx

```diff
@@ -195,7 +194,7 @@ describe('Testing AdmitPatientForm', () => {
     expect(mockedAssignPatientToBed).toHaveBeenCalledWith(3, mockPatientAlice.uuid, 'encounter-uuid');
     expect(mockedShowSnackbar).toHaveBeenCalledWith({
       kind: 'success',
-      subtitle: '{{patientName}} has been successfully admitted and assigned to bed bed3',
+      subtitle: 'Alice Johnson has been successfully admitted and assigned to bed bed3',
```

Comment:
This is owing to the interpolation syntax changes in the React i18next mock that align it much closer to the actual implementation.